akka-actor/src/main/scala/akka/actor/ActorLogMarker.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor
+
+import akka.annotation.ApiMayChange
+import akka.event.LogMarker
+
+@ApiMayChange
+object ActorLogMarker {
```

Comment:
Mark it `@InternalApi` because even if it is interesting for users on the consuming side it is not really for user interaction on this side.

Reply:
One purpose was to use this for documentation of the marker names and properties used, in apidoc. Somewhat strange to make it InternalApi if shown in apidoc?

Reply:
Can we add a Scaladoc note that they are for doc purposes and not for coding against instead then? I don't think we want to promise bincomp. for the factory methods?

Reply:
added note, see 07bd8a0f43